Sources/MapboxDirections/Extensions/Codable.swift

```diff
@@ -28,12 +28,18 @@ enum PolyLineString {
 }
 
 extension PolyLineString: Codable {
+    private enum LineStringCodingKeys: String, CodingKey {
+        case coordinates
+    }
+    
     init(from decoder: Decoder) throws {
         let container = try decoder.singleValueContainer()
         let options = decoder.userInfo[.options] as? DirectionsOptions
         switch options?.shapeFormat ?? .default {
         case .geoJSON:
-            self = .lineString(try container.decode(LineString.self))
+            let lineStringContainer = try decoder.container(keyedBy: LineStringCodingKeys.self)
+            let coordinates = try lineStringContainer.decode([CLLocationCoordinate2DCodable].self, forKey: .coordinates).map { $0.decodedCoordinates }
```

Comment:
Is it necessary to manually decode the LineString here? Can we continue to rely on Turf’s implementation of LineString’s Codable conformance?

Reply:
`Turf` does not provide `LineString` `Codable` conformance. So I decided we should not add it in `Directions` either.

Reply:
Ah, that’s a little bit awkward. But Geometry is codable, so we can continue to wrap the LineString in a Geometry and decode that, right?

Reply:
Yes, I think it could work for decoding, but encoding `Geometry` will also serialize it's `type`, which is undesired.